.github/workflows/test.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.61.0 # MSRV - keep in sync with what rustls considers MSRV
+          - 1.64.0 # MSRV - keep in sync with what rustls considers MSRV
```

Comment:
I think this comment is a bit inaccurate now, the MSRV of Rustls is still 1.61, it's the MSRV of rustls-platform-verifier we're having to match now.

Perhaps best to reword the comment to say "keep in sync with what rustls and rustls-platform-verifier consider MSRV"?